build.gradle.kts

```diff
@@ -17,8 +17,9 @@ ihmc {
 app.entrypoint("SlaveInfo", "us.ihmc.etherCAT.master.SlaveInfo")
 
 mainDependencies {
-   api("us.ihmc:soem:1.4.0-ihmc1")
-   api("us.ihmc:soem-platform-linux:1.4.0-ihmc1")
+   api("us.ihmc:soem:1.5.0")
+   api("us.ihmc:soem-platform-linux-x86_64:1.5.0")
+   api("us.ihmc:soem-platform-linux-arm64:1.5.0")
```

Comment:
Will have to publish these

Reply:
Looks good. Looks like that's why the checks are failing. Maybe you should go ahead and publish so the tests in the PR pass?